functions/Set-DbaMaxMemory.ps1

```diff
@@ -57,28 +60,25 @@ function Set-DbaMaxMemory {
     .EXAMPLE
         PS C:\> Get-DbaCmsRegServer -SqlInstance sqlserver | Test-DbaMaxMemory | Where-Object { $_.SqlMaxMB -gt $_.TotalMB } | Set-DbaMaxMemory
 
-        Find all servers in SQL Server Central Management server that have Max SQL memory set to higher than the total memory
+        Find all servers in SQL Server Central Management Server that have Max SQL memory set to higher than the total memory
         of the server (think 2147483647), then pipe those to Set-DbaMaxMemory and use the default recommendation.
 
 #>
     [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
     param (
-        [Parameter(Position = 0)]
+        [Parameter (Mandatory = $false)]
         [Alias("ServerInstance", "SqlServer", "SqlServers", "ComputerName")]
         [DbaInstanceParameter[]]$SqlInstance,
         [Alias("Credential")]
         [PSCredential]$SqlCredential,
-        [Parameter(Position = 1)]
+        [Parameter (Mandatory = $false)]
         [int]$MaxMB,
+        [Parameter(ValueFromPipeline = $True)]
+		[PSCustomObject[]]$Collection,
```

Comment:
We accidentally named stuff wrong back in the day. All/most piped input should be `$InputObject` and `[Parameter(ValueFromPipeline = $True)]` can be plain ol `[Parameter(ValueFromPipeline)]`